bin/update

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+echo "Updating dependencies:"
+yarn
+
+echo "Updating client dependencies:"
```

Comment:
@AleksSenkou Can you please explain why it updates client dependencies twice, but never updates server dependencies? I just don't get it :blush:

Reply:
@nikitasakov ha, good catch 👍 

Will change it in a separate PR